org.lflang/src/lib/core/federated/RTI/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+# This is a cmake build script providing a solution for compiling
+# the RTI in this directory..
+#
+# Usage:
+#
+# To compile with cmake, run the following commands:
+#
+# $> mkdir build && cd build
+# $> cmake ../
+# $> make
+# $> sudo make install
+#
+# This create a binary RTI in the current working directory. Please put this in
```

Comment:
I am not sure in which context this cmake file is called. By default, cmake installs to the system root. Only if an install prefix is given when running cmake, it alternates the installation path. This is analogous to th way `configure` works. In case of the C++ LF target, the installation path is automatically adjusted to install in the project dir (or cwd if there is no src directory). However, for the RTI you might whant to have it available in the system root path.

Reply:
The cmake command is invoked by the user manually in a `build` directory in this folder. Just running `make` will produce an RTI binary in that folder (i.e., `org.lflang/src/lib/core/federated/RTI/build/RTI`). 

As @cmnrd pointed out, I think we need the `RTI` to be accessible from everywhere. Therefore, by default, `sudo make install` will put the binary in `/usr/local/bin/`. This behavior can always be overridden by passing the [CMAKE_INSTALL_PREFIX](https://cmake.org/cmake/help/v3.0/variable/CMAKE_INSTALL_PREFIX.html) variable to cmake (e.g., `cmake -DCMAKE_INSTALL_PREFIX=/usr ../`). This is I think standard CMake procedure as best as I can tell. The default installation place is documented as "`/usr/local` on UNIX and `c:/Program Files` on Windows".